src/components/Footer/Footer.js

```diff
@@ -0,0 +1,147 @@
+  constructor(props) {
+    super(props);
+    this.state = { id: uuid4() };
+  }
+
+  componentDidMount() {
+    const buttonId = `${this.state.id}-language-dropdown-button`;
+    const initialize = setInterval(() => {
+      if (window.chi && document.getElementById(buttonId)) {
+        window.chi.dropdown(document.getElementById(buttonId));
+        clearInterval(initialize);
+      }
+    }, 1000);
+  }
+
+  render() {
+    const { id } = this.state;
+    const buttonId = `${id}-language-dropdown-button`;
+    const isTabletSize = this.props.footerSize === 'Portrait tablet';
+    const isDesktopSize = this.props.footerSize === 'Desktop';
+    const footerStyle = isDesktopSize
+      ? { minWidth: '1256px' }
+      : { width: isTabletSize ? '736px' : '368px' };
+    const footerInternalContentStyle = isDesktopSize ? '-flex--row' : '-flex--column';
+    const ulElementStyle = isDesktopSize ? '-flex--row' : '-flex--column';
+    const links = [
+      { url: 'https://www.lumen.com/en-us/about.html', text: 'About Us', newTab: false },
+      { url: 'https://www.centurylink.com/aboutus/community/community-development/programs-for-customers-with-disabilities.html', text: 'Accessibility', newTab: false },
+      { url: 'https://jobs.lumen.com', text: 'Careers', newTab: true },
+      { url: 'https://www.lumen.com/en-us/contact-us.html', text: 'Contact Us', newTab: false },
+      { url: '#', text: 'Cookie Settings', newTab: false },
+      { url: 'https://www.centurylink.com/aboutus/legal.html', text: 'Legal', newTab: true },
+      { url: 'https://www.lumen.com/en-us/about/legal/acceptable-use-policy.html', text: 'Legal Notices', newTab: true },
+      { url: 'https://www.lumen.com/en-us/about/legal/privacy-notice.html', text: 'Privacy', newTab: false },
+      { url: 'https://www.centurylink.com/aboutus/company-information/public-policy.html', text: 'Public Policy', newTab: true },
+      { url: 'https://www.lumen.com/help/en-us/home.html', text: 'Support', newTab: true },
+      { url: 'https://www.lumen.com/en-us/about/legal/commission-for-complaints-for-telecom-television-services-ccts', text: 'Canadian Ombudsman', newTab: false },
+      { url: 'https://assets.lumen.com/is/content/Lumen/gender-pay-gap-report?Creativeid=b565bf28-6d55-4f86-a245-2f0eb40b91a3', text: 'Gender Pay Gap 2020 (UK)', newTab: false },
+      { url: 'https://www.lumen.com/es-ar/about/legal/latam-tariffs-regulatory.html', text: 'Legal (Latin America)', newTab: false },
+      { url: 'https://www.lumen.com/en-us/about/legal/de-legal-notices.html', text: 'Legal Notices (Germany)', newTab: false },
+      { url: 'https://assets.lumen.com/is/content/Lumen/modern-slavery-statement-1?Creativeid=5dbb9687-e4a8-4a40-ae00-673ce51d6a80', text: 'Modern Slavery Statement 2020 (UK)', newTab: false },
+      { url: 'https://www.lumen.com/en-us/about/legal/uk-tax-strategy.html', text: 'UK Tax Strategy', newTab: false },
+    ];
+
+    return (
+      <>
```

Comment:
Why do we need `<>` if there is only one element inside?

Reply:
Removed